src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/CostsService.java

```diff
@@ -1,39 +1,88 @@
 package uk.gov.companieshouse.overseasentitiesapi.service;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.api.model.payment.Cost;
 
 import java.util.Collections;
+import uk.gov.companieshouse.overseasentitiesapi.exception.SubmissionNotFoundException;
 
 @Service
 public class CostsService {
 
-    private static final String COST_DESC = "Register Overseas Entity fee";
+    private static final String REGISTER_COST_DESC = "Register Overseas Entity fee";
```

Comment:
Would `REGISTER_COST_DESCRIPTION` seem more readable instead of `REGISTER_COST_DESC`? Same with the other variable names that mention `DESC`. Just a minor thing but in my head `DESC` sounds like `DESCENDING`.

Reply:
Happy if `DESC` was changed to `DESCRIPTION` - would make code more readable. Private vars so should be minimal impact.